lib/logstash/outputs/lumberjack.rb

```diff
@@ -73,6 +74,10 @@ def flush(events, close = false)
       retry
     end # begin
   end
+
+  def close
+    buffer_flush(:force => true) 
```

Comment:
maybe calling the buffer_flush in the receive if the event is a shutdown would be better, this was the close  could just call close on the `@client`?